cea/interfaces/dashboard/tools/templates/folder_listing.html

```diff
@@ -16,6 +16,15 @@ <h4 class="modal-title" id="cea-folder-dialog-title">{{title}}</h4>
     </ol>
   </nav>
 
+  {% if drives %}
+  <select class="form-control cea-drive-select" id="cea-drive-select"
+          onchange="folder_dialog_navigate_to('{{fqname}}', $(this).val(), '.')">
+    {% for drive in drives %}
+    <option value="{{ drive.lower() }}" {% if breadcrumbs[0][:2].lower() == drive.lower() %} selected {% endif %}>{{ drive }}</option>
```

Comment:
Could change the drive to include the trailing slash i.e. `C:\` since `os.path.abspath` changes the input `c:.` to the working directory

```suggestion
    <option value="{{ drive }}\" {% if breadcrumbs[0][:2].lower() == drive.lower() %} selected {% endif %}>{{ drive }}</option>
```

Reply:
@daren-thomas I appreciate if you can check this...